cli/src/time_util.rs

```diff
@@ -1,97 +1,37 @@
-use chrono::format::StrftimeItems;
-use chrono::DateTime;
-use chrono::FixedOffset;
-use chrono::LocalResult;
-use chrono::TimeZone as _;
-use chrono::Utc;
+use jiff::fmt::strtime;
+use jiff::tz::Offset;
+use jiff::tz::TimeZone;
+use jiff::Zoned;
 use jj_lib::backend::Timestamp;
-use once_cell::sync::Lazy;
-use thiserror::Error;
 
-/// Parsed formatting items which should never contain an error.
-#[derive(Clone, Debug, Eq, PartialEq)]
-pub struct FormattingItems<'a> {
-    items: Vec<chrono::format::Item<'a>>,
+fn datetime_from_timestamp(context: &Timestamp) -> Result<Zoned, jiff::Error> {
+    Ok(
+        jiff::Timestamp::from_millisecond(context.timestamp.0)?.to_zoned(TimeZone::fixed(
+            Offset::constant((context.tz_offset / 60).try_into().unwrap_or_default()),
```

Comment:
nit: instead of falling back to `0`, maybe we can clamp by `i8::MIN`/`MAX`. Then jiff will report overflow error.